packages/select/src/select.tsx

```diff
@@ -30,6 +31,7 @@ export const SelectField = forwardRef<SelectFieldProps, "select">(
         {...ownProps}
         ref={ref}
         className={cx("chakra-select", className)}
+        bg={useColorModeValue("white", "gray.700")}
```

Comment:
Please move this change to the [select's theme](https://github.com/chakra-ui/chakra-ui/blob/main/packages/theme/src/components/select.ts)

Reply:
Hi, Ive had a look into this, but due to the nature of the select component I am struggling to inspect the html associated with it. Also I think this might be a different problem to what I originally thought, as the background of the Select bar itself is meant to be transparent as on chrome but on firefox for some reason it has extra space above and below the options which show the white background of the default html page.